contracts/multisig/src/msg.rs

```diff
@@ -38,7 +38,7 @@ pub enum QueryMsg {
     #[returns(Multisig)]
     GetMultisig { session_id: Uint64 },
 
-    #[returns(Key)]
```

Comment:
Do you know why Github keeps displaying these warnings, including the one below?

Reply:
I tried researching about this when I was solving it but got no lead. My assumption is that because the module is only being used as a datatype (and not actually being initialized or used as a variable for example), then it doesn't actually use anything from the library itself.